docker/DOIRootCA2.crt

```diff
@@ -0,0 +1,56 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This file isn't meant to be unreadable. It is necessary to include it because it is the public key for DOI's "SSL Intercept" system. It allows the user to access the web when VPN is connected.